pkg/mounter/mounter-rclone.go

```diff
@@ -191,8 +191,8 @@ func (rclone *RcloneMounter) Mount(source string, target string) error {
 		return err
 	}
 
-	if rclone.ObjPath != "" {
-		bucketName = fmt.Sprintf("%s:%s/%s", remote, rclone.BucketName, rclone.ObjPath)
+	if rclone.objectPath != "" {
+		bucketName = fmt.Sprintf("%s:%s/%s", remote, rclone.BucketName, rclone.objectPath)
```

Comment:
Rclone accepts bucket name in the format `remote:bucket/path/to/dir`. Hence you need to add similar logic to trim `/` prefix if it exists.